src/backend/tty.rs

```diff
@@ -496,6 +497,19 @@ impl Tty {
     ) -> anyhow::Result<()> {
         debug!("device added: {device_id} {path:?}");
 
+        // skip /dev/dri/render* node
+        match path.as_str() {
+            Ok(str) => {
+                if str.contains("render") {
+                    return Ok(());
+                }
+            }
+            Err(err) => {
+                error!("invalid path: {}", err);
+                return Ok(());
+            }
+        }
```

Comment:
```suggestion
        if let Some(path_os_str) = path.file_name() and let Some(path_str) = path_os_str.to_str() {
        	if path_os_str.contains("render") {
        		do_things();
        	} else {
        		do_things();
        	}
        }
```

Isnt it better to do it this way instead of blindly checking if the whole string contains the word `render`?

Reply:
@HigherOrderLogic the udev monitor only listens for drm events. So I think that makes the possible file paths 
`/dev/dri/card*` and `/dev/dri/render*` only.

Smithay:
`backend/udev.rs: let monitor = MonitorBuilder::new()?.match_subsystem("drm")?.listen()?;`

I can make the changes if strict filtering is needed.